src/authProviderHelper.ts

```diff
@@ -6,6 +6,11 @@ class AuthProviderHelper {
     this.initalized = true;
   }
 
+
```

Comment:
Prettier may not like this extra-line. I'll remove it to avoid unnecessary changes/commits later